### PR TITLE
Add pointer to monitoring-core.yaml

### PR DESCRIPTION
**Pull request by duglin**
DO NOT merge this until https://github.com/knative/serving/pull/6663
is merged

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by mattmoor** [comment]
/hold

(per the PR description)

**Comment by mattmoor** [comment]
/lgtm
/assign @samodell 

The linked PR has been stamped, and should hopefully merge soon.

**Comment by duglin** [comment]
knative/serving#6663 is now merged.

File: docs/install/knative-with-any-k8s.md [hunk 1]
```diff
@@ -496,6 +496,12 @@ You can find a number of sample on how to get started with Knative Eventing [her
 
 Knative provides a bundle of monitoring components that can be used to make the Serving and Eventing components more observable.
 
+Before you install any specific monitoring components, you must first install the core monitoring pieces:
+
+```bash
+kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-core.yaml
```
Comment (mattmoor) [inline docs/install/knative-with-any-k8s.md]:
```suggestion
kubectl apply --filename {{< artifact repo="serving" file="monitoring-core.yaml" >}}
```

This is something that landed in parallel to your change, which lets us use nightly builds on the pre-release page.

Comment (duglin) [reply docs/install/knative-with-any-k8s.md]: done!

**Pull request merged**